source/extensions/filters/http/header_to_metadata/header_to_metadata_filter.cc

```diff
@@ -120,7 +164,7 @@ bool HeaderToMetadataFilter::addMetadata(StructMap& map, const std::string& meta
     return false;
   }
 
-  std::string decodedValue = std::string(value);
+  std::string decodedValue = value;
```

Comment:
Now that `value` is an std::string that this method owns we can just use that directly, without copying it into `decodedValue` first.